README.md

```diff
@@ -158,3 +185,25 @@ Troubleshooting
 ### My requests are being rate limited
 
 Set the `rateLimitToken` property, and make sure the token is valid for the environment you're testing.  These tokens live in the encrypted hieradata in [govuk-secrets](https://github.com/alphagov/govuk-secrets).
+
+### Authentication issues
+- Ensure the right permissions have been set for your test user
+- Disable 2FA
+
+You may need to use a different variable if `USERNAME` does not keep the user details you have set. Update this in [signon.scala](https://github.com/alphagov/govuk-load-testing/blob/master/src/test/scala/govuk/Signon.scala#L30
+)
+
+NOTE: `USERNAME` needs to be an email address
+
+### No such file or directory
+If you see an error such as:
+```
+15:16:48.524 [ERROR] i.g.a.Gatling$ - Run crashed
+java.io.FileNotFoundException: test-data/lorem-ipsum.txt (No such file or directory)
+```
+
+Ensure you run gatling in this repo:
```

Comment:
should we say `the govuk-load-testing repo` just to be clear?

Reply:
Good point @edwardkerry , have just updated this 👍